### PR TITLE
Nth term of gp

### LINKED ISSUE #6122: Nth term of GP in Python
## 🚀 Feature

(A clear and concise description of what the feature is.)

### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#reporting-new-issues)?

YES

I would like to add the code for finding Nth term of GP  in python

### PR DESCRIPTION
**Pull request by arpit456jain**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?
YES
### Description
I have  add the code for finding Nth term of GP in python

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

Fixes : #6122

### CODE REVIEW

File: Python/math/nth_term_of_GP.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import math
+'''
+This is a simple code in python to find the Nth term of a GP , let say 1 3 9 27 is GP its first term (a) is 1 and the Common Ratio (r) is 3 and if n=3 it means 3rd term which is 9.
```
Comment (atarax665) [inline Python/math/nth_term_of_GP.py]: Justify align this and move it to the top.

File: Python/math/nth_term_of_GP.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+Enter the value of first term a and the Common Ratio r of a GP
+1 3
+Enter the value of N to find the value of its Nth term
+3
+
+Output:
+The Nth term of this GP is 9
+Time Complexity : O(1)
+Space Complexity : O(1)
+'''
```
Comment (atarax665) [inline Python/math/nth_term_of_GP.py]: Add a line at the EOF.

File: Python/math/nth_term_of_GP.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+def termOfGP(A, B, N):
+    r = B / A
+    ans = A * ((math.pow(r, N - 1)))
+    return ans
+
+
+
+def main():
+    print("Enter the value of first term a and the Common Common Ratio r of a GP")
```
Comment (atarax665) [inline Python/math/nth_term_of_GP.py]:
```suggestion
    print("Enter the value of the first term 'a' and the Common Common Ratio 'r' of a GP")
```

File: Python/math/nth_term_of_GP.py [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+
+#  Driver Code Starts
+if __name__ == "__main__":
+    main()
+
+'''
+Sample Input/Output:
+
+Enter the value of first term a and the Common Ratio r of a GP
```
Comment (atarax665) [inline Python/math/nth_term_of_GP.py]:
```suggestion
Enter the value of the first term 'a' and the Common Ratio 'r' of a GP
```

**Review by atarax665 — CHANGES_REQUESTED** [review]

File: Python/math/nth_term_of_GP.py (hunk 4 above)
Comment (arpit456jain) [reply Python/math/nth_term_of_GP.py]: done

File: Python/math/nth_term_of_GP.py (hunk 3 above)
Comment (arpit456jain) [reply Python/math/nth_term_of_GP.py]: done

File: Python/math/nth_term_of_GP.py (hunk 2 above)
Comment (arpit456jain) [reply Python/math/nth_term_of_GP.py]: done

File: Python/math/nth_term_of_GP.py (hunk 1 above)
Comment (arpit456jain) [reply Python/math/nth_term_of_GP.py]: i have moved the statement to the top but  i don't understand justify align

**Comment by arpit456jain** [comment]
@atarax665 all requested are done except the justify align please brief it little bit

**Comment by arpit456jain** [comment]
@atarax665 kindly review my PR

**Review by ankitaggarwal23 — COMMENTED** [review]
Don't accept concept based problem

**Pull request closed without merging**